src/Routing/RouteProcessor.php

```diff
@@ -98,7 +98,7 @@ protected function _processHandler(Context $c, $handler, &$response): bool
       return true;
     }
 
-    if($handler)
+    if($handler || is_numeric($handler))
```

Comment:
whats the use case for this?

Reply:
If the handler function just returns 0 then this wont be hit, not setting the response